src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/errors/ErrorNotificationRequest.java

```diff
@@ -22,6 +22,9 @@
     @JsonProperty("reference_id")
     public final String referenceId;
 
+    @JsonProperty("service")
+    public final String service;
```

Comment:
is this the model that's sent to the external supplier? if so, then we don't want this field here.